src/App.tsx

```diff
@@ -15,71 +15,78 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type Props = {
+  select: string,
+  name: string,
+  switchSelect: (name: string) => void
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+const Element: React.FC<Props> = ({ select, name, switchSelect }) => {
+  const [check, setCheck] = useState<boolean>(select === name);
```

Comment:
do we actually need useState here, we cannot instead of using `check` variable just check if `select === name`?